travis-ci.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Don't fail silently when a step doesn't succeed
+set -e
+
+# if [ -z "$TRAVIS" ]; then
+#     echo "this script is intended to be run only on travis" >&2
+#     exit 1
+# fi
+
+make container deploy_kind
+
+echo "|---- Creating new Sonobuoy run/waiting for results..."
+./sonobuoy run --image-pull-policy=IfNotPresent -m Quick --wait & PID=$!
+
+# While process is running...
+while kill -0 $PID 2> /dev/null; do 
+    printf  "."
+    sleep 10
+done
+
+echo "|---- Retrieving results..."
+./sonobuoy retrieve
+
+echo "|---- Results found, validating output..."
+for i in *.gz; do
+mkdir -p results
+tar -xvf $i -C results
+grep -q "Test Suite Passed" results/plugins/e2e/results/e2e.log
```

Comment:
This section for getting and validating output can be reduced to something like:
```
outFile=$(./sonobuoy retrieve)
results=$(./sonobuoy e2e $outFile)
echo $results
exitCode=$($results | grep 0)
exit $exitCode
```

This is of course based on the idea that https://github.com/heptio/sonobuoy/pull/598 merges (any minute after tests go green).

That has the benefit that we also are 
1) dogfooding our own command
2) listing the failed tests, not just reporting failure or not (I know right now this is quick mode so its just one test, but you get the point)
3) less coupled to the text output by ginkgo for passed/failed
4) avoid some of the file creation/manipulation that can sometimes (in my experience) have odd edge cases

Reply:
We need to look at `sonobuoy e2e outfile`, it's not super helpful which is why I avoided (and your PR was still in flight).